stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -82,6 +82,23 @@ static CFRange cast(_swift_shims_CFRange value) {
   return cast(CFStringCreateWithSubstring(cast(alloc), cast(str), cast(range)));
 }
 
+_swift_shims_CFComparisonResult
+swift::_swift_stdlib_CFStringCompare(
+                              _swift_shims_CFStringRef string,
+                              _swift_shims_CFStringRef string2) {
+  return cast(CFStringCompareWithOptionsAndLocale(cast(string),
+                                                  cast(string2),
+                                                  { 0, CFStringGetLength(cast(string)) },
+                                                  0,
+                                                  NULL));
+}
+
+__swift_uint8_t
+swift::_swift_stdlib_isNSString(id obj) {
+  //TODO: use _NSIsNSString here
```

Comment:
_NSIsNString wasn't called that on older OSs, so I need to work out a backwards deployment story before we can do that. Checking the type id is fine for now.